data/formats-data.js

```diff
@@ -3852,7 +3859,7 @@ exports.BattleFormatsData = {
 		tier: "PU",
 	},
 	buizel: {
-		randomBattleMoves: ["waterfall", "aquajet", "switcheroo", "brickbreak", "bulkup", "batonpass", "icepunch"],
+		randomBattleMoves: ["waterfall", "aquajet", "switcheroo", "brickbreak", "bulkup", "batonpass", "icepunch", "hydropump", "pursuit"],
```

Comment:
the original set was fine...

Reply:
mix pump sounds considerably good and the threat of setup/bp makes that pursuit even tastier

Reply:
the only thing pursuit could any use for would be hopefully trapping gastly, since this is one speed tier above gastly.

if they're going to switch out, wouldnt you want to get the boost then pass when they bring in something threatening?

Reply:
oh that's fair